tests/integration/fuzzing/strategies_test.cairo

```diff
@@ -0,0 +1,50 @@
+%lang starknet
+
+from starkware.cairo.common.math import assert_le
+
+@external
+func test_integers{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt):
+    %{
+        given(
+            a = strategies.integers(10, 20),
```

Comment:
Singular form sounds more natural to me.
`given strategy integers`
`given strategies integers`

Reply:
hmm makes sense 👍🏻 renaming